### PR TITLE
[15.0] [FIX] purchase_request: invalid field

### PR DESCRIPTION
**Pull request by florentx**

### CODE REVIEW

**Review by ajaniszewska-dev — APPROVED** [review]

File: purchase_request/models/stock_move.py [hunk 1]
```diff
@@ -75,7 +75,7 @@ def _action_cancel(self):
     def _compute_purchase_request_ids(self):
         for rec in self:
             rec.purchase_request_ids = rec.purchase_request_allocation_ids.mapped(
```
Comment (simahawk) [inline purchase_request/models/stock_move.py]: you could also drop "mapped"

**Review by simahawk — APPROVED** [review]

File: purchase_request/models/stock_move.py (hunk 1 above)
Comment (florentx) [reply purchase_request/models/stock_move.py]: how?

Comment (simahawk) [reply purchase_request/models/stock_move.py]: just remove it: `rec.purchase_request_allocation_ids.purchase_request_line_id.request_id`.
Works fine w/ relations.

Comment (florentx) [reply purchase_request/models/stock_move.py]: I didn't know that this works for o2m and m2m fields.
I still wonder if `mapped` has a benefit to remove duplicates before returning the recordset?

**Pull request merged**